libraries/AP_HAL_SITL/HAL_SITL_Class.cpp

```diff
@@ -165,6 +165,11 @@ void HAL_SITL::setup_signal_handlers() const
     sa.sa_flags = SA_NOCLDSTOP;
     sa.sa_handler = HAL_SITL::exit_signal_handler;
     sigaction(SIGTERM, &sa, NULL);
+    sigaction(SIGINT, &sa, NULL);
+    sigaction(SIGKILL, &sa, NULL);
+    sigaction(SIGHUP, &sa, NULL);
+    sigaction(SIGQUIT, &sa, NULL);
```

Comment:
You shouldn't be able to catch `KILL`.

Since we're doing this independent of running under coverage, this is going to need testing on WSL, WSL2, MacOSX and cygwin

Reply:
fixed with a define